react-app/nginx.conf

```diff
@@ -1,4 +1,26 @@
 server {
+  # Enable HTTP Strict Transport Security (HSTS) to force clients to always
+  # connect via HTTPS (do not use if only testing)
+  add_header Strict-Transport-Security "max-age=31536000;";
```

Comment:
Not sure how this still works with localhost. Doesn't seem to impede it, but it's not https there.